src/main/java/Course.java

```diff
@@ -0,0 +1,30 @@
+import java.util.ArrayList;
+
+// entity layer
+
+public class Course {
+    /*
```

Comment:
Use Javadoc comments instead of regular multiline comments for classes and methods

Reply:
fixed